gp-populate-anything/gppa-colorpicker-choice-template.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Gravity Perks // Populate Anything // Add Custom Template Row for Color Picker
+ * https://gravitywiz.com/documentation/gravity-forms-populate-anything/
+ * 
+ * Instructions Video: https://www.loom.com/share/c062a781a86242de9f1ebb92492a408c
+ *
+ * Adds "Color Picker" template row to the Populate Anything interface and processes its value (with Jet Sloth's Color Picker plugin).
+ *
+ * Plugin Name:  GP Populate Anything — Color Picker Choice Template
+ * Plugin URI:   https://gravitywiz.com/documentation/gravity-forms-populate-anything/
+ * Description:  Adds a custom template row and value processing for "Color Picker" in Populate Anything.
+ * Author:       Gravity Wiz
+ * Version:      1.0
+ */
+class GPPA_Color_Picker_Template {
```

Comment:
Can this be named similar to [GPPA_Compatibility_JetSloth_Image_Choices](https://github.com/gravitywiz/snippet-library/pull/1072)

```suggestion
class GPPA_Compatibility_JetSloth_Color_Picker_Template {
```